std/assembly/string.ts

```diff
@@ -552,7 +558,8 @@ import { idof } from "./builtins";
     var len = <usize>this.length;
     if (!len) return this;
     var codes = __alloc(len * 3 * 2, idof<String>());
-    var specialsUpperLen = SPECIALS_UPPER.length;
+    var specialsPtr = changetype<usize>(SPECIALS_UPPER);
```

Comment:
Does it help to have it in a local? Asking because the pointer is inlined anyway.

Reply:
Is it slowdown parsing?